sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -12,6 +12,18 @@
 
 ### Other Changes
 
+## 5.13.0-beta.1 (2022-08-01)
+
+### Features Added
+
+- Added algorithm for mapping partition keys to partition ids.
+- Added EventHubBufferedProducerAsyncClient and EventHubBufferedProducerClient
+
+### Bugs Fixed
+
+- Introducing ReactorShim to proxy certain reactive operations to appropriate Reactor operators, these are the operations for which recent Reactor versions have more optimized operators compared to an older version, or same operators with breaking change across Reactor versions
```

Comment:
Does `ReactorShim` need to be called out at all given it's an implementation detail

Reply:
Hm... not sure... But it is a bug fix that many people are curious about since it deals with windowsTimeout.